### PR TITLE
Plot finished DVCLive only experiments

### LINKED ISSUE #4491: After experiment is done it's not select for plot anymore
It stays on the workspace. It means when I run a new experiment the previous plotted one just disappears underneath me.

### PR DESCRIPTION
**Pull request by mattseddon**
# 2/2 `main` <- #4565 <- this  

Closes #4491

WIP

### Demo (using https://github.com/iterative/dvclive/pull/674)

https://github.com/iterative/vscode-dvc/assets/37993418/b63f9ea8-d997-4c90-8604-e2e0db20588f

https://github.com/iterative/vscode-dvc/assets/37993418/8c9286ff-8d3e-4834-843f-37a69c59244a

### CODE REVIEW

File: extension/src/experiments/index.ts [hunk 1]
```diff
@@ -589,6 +593,15 @@ export class Experiments extends BaseRepository<TableData> {
     return this.data.update()
   }
 
+  public checkDvcLiveOnly(fetched: string[]) {
```
Comment (mattseddon) [inline extension/src/experiments/index.ts:596]: [I] Better name

File: extension/src/experiments/model/status/collect.ts [hunk 2]
```diff
@@ -10,19 +10,28 @@ import { hasKey } from '../../../util/object'
...
+  acc: ColoredStatus,
+  experiment: Experiment,
+  dvcLiveOnlyExpName: string | undefined
+): void => {
   const { id, executorStatus: status } = experiment
   if (!id || isQueued(status) || hasKey(acc, id)) {
     return
   }
 
+  if (acc[EXPERIMENT_WORKSPACE_ID] && id === dvcLiveOnlyExpName) {
```
Comment (mattseddon) [inline extension/src/experiments/model/status/collect.ts:30]: [I] Breadcrumbs

File: extension/src/fileSystem/index.ts [hunk 3]
```diff
@@ -269,6 +279,21 @@ export const getPidFromFile = async (
   return pid
 }
 
+export const getDetailFromFile = (
```
Comment (mattseddon) [inline extension/src/fileSystem/index.ts:282]: [I] Name

File: extension/src/plots/index.ts [hunk 4]
```diff
@@ -189,7 +189,17 @@ export class Plots extends BaseRepository<TPlotsData> {
   private async sendPlots() {
     await this.isReady()
 
-    return this.webviewMessages.sendWebviewMessage()
+    await this.webviewMessages.sendWebviewMessage()
+
+    const fetchedRevs = []
```
Comment (mattseddon) [inline extension/src/plots/index.ts:194]: [I] Breadcrumbs => combine into well named function

**Review by julieg18 — APPROVED** [review]

**Pull request merged**